centreon/doc/API/v23.10/Configuration/ServiceTemplate/Schema/ServiceTemplate.yaml

```diff
@@ -0,0 +1,256 @@
+    example: null
+  check_command_id:
+    type: integer
+    description: "Check command ID."
+    minimum: 1
+    nullable: true
+    example: null
+  check_command_args:
+    type: array
+    items:
+      type: string
+      description: "Arguments of the check command."
+      example: ["80", "90"]
+  check_timeperiod_id:
+    type: integer
+    description: "Time period ID of the check command."
+    minimum: 1
+    nullable: true
+    example: null
+  max_check_attempts:
+    type: integer
+    description: "Define the number of times that monitoring engine will retry the service check command if it returns any non-OK state."
+    nullable: true
+  normal_check_interval:
+    type: integer
+    description: |
+      Define the number of 'time units' between regularly scheduled checks of the service.
+      
+      With the default time unit of 60s, this number will mean multiples of 1 minute.
+    nullable: true
+  retry_check_interval:
+    type: integer
+    description: |
+      Define the number of "time units" to wait before scheduling a re-check for this service after a non-UP state was detected.
+      
+      With the default time unit of 60s, this number will mean multiples of 1 minute.
+      
+      Once the service has been retried max_check_attempts times without a change in its status,
+      it will revert to being scheduled at its "normal" check interval rate.
+    nullable: true
+  active_check_enabled:
+    type: integer
+    description: |
+      Indicates whether the active checks is enabled or not
+      
+      * `0` - STATUS_DISABLED
+      * `1` - STATUS_ENABLE
+      * `2` - STATUS_DEFAULT (inheritance of his parent's value. If there is no parent, the values used will be that of the Centreon Engine)
+    example: 2
+  passive_check_enabled:
+    type: integer
+    description: |
+      Indicates whether the passive checks is enabled or not
+      
+      * `0` - STATUS_DISABLED
+      * `1` - STATUS_ENABLE
+      * `2` - STATUS_DEFAULT (inheritance of his parent's value. If there is no parent, the values used will be that of the Centreon Engine)
+    example: 2
+  volatility_enabled:
+    type: integer
+    description: |
+      Indicates whether the service is "volatile" or not
+
+      * `0` - STATUS_DISABLED
+      * `1` - STATUS_ENABLE
+      * `2` - STATUS_DEFAULT (inheritance of his parent's value. If there is no parent, the values used will be that of the Centreon Engine)
+    example: 2
+  notification_enabled:
+    type: integer
+    description: |
+      Specify whether notifications are enabled or not
+      
+      * `0` - STATUS_DISABLED
+      * `1` - STATUS_ENABLE
+      * `2` - STATUS_DEFAULT (inheritance of his parent's value. If there is no parent, the values used will be that of the Centreon Engine)
+    example: 2
+  is_contact_additive_inheritance:
+    type: boolean
+    description: |
+      Only used when notification inheritance for hosts and services is set to vertical inheritance only.
```

Comment:
the label in the GUI is "vertical inheritance only"?

Reply:
@cg-tw Yes